discovery_child_development/pipeline/labelling/taxonomy/prompts/taxonomy_categories.json

```diff
@@ -0,0 +1,57 @@
+{
+  "development": {
+    "Cognitive development": "cognition, memory, problem-solving, executive function",
+    "SEND": "special needs, disabilities, learning difficulties",
```

Comment:
You could add "autism, dyslexia, attention-deficit hyperactivity disorder" as those might be quite popular

I was also thinking about "speech and language therapy" but such data will probably be captured by the communication and language category already - so I'd say no need to add.